projects/components/src/table/table.component.ts

```diff
@@ -342,6 +342,9 @@ export class TableComponent
   @Input()
   public selectionMode?: TableSelectionMode = TableSelectionMode.Single;
 
+  @Input()
+  public clickOnRowToSelect?: boolean = true;
```

Comment:
We have `TableSelectionMode.None` but we're not using that anywhere in the table component.
can we use that instead of creating a new component?